.about.yml

```diff
@@ -0,0 +1,99 @@
+---
+# .about.yml project metadata
+#
+# Short name that acts as the project identifier (required)
+name: federalist
+
+# Full proper name of the project (required)
+full_name: Federalist
+
+# The type of content in the repo
+# values: app, docs, policy
+type: app
+
+# Describes whether a project team, working group/guild, etc. owns the repo (required)
+# values: guild, working-group, project
+owner_type: project
+
+# Name of the main project repo if this is a sub-repo; name of the working group/guild repo if this is a working group/guild subproject
+parent: federalist
+
+# Maturity stage of the project (required)
+# values: discovery, alpha, beta, live
+stage: alpha
+
+# Whether or not the project is actively maintained (required)
+# values: active, deprecated
+status: active
+
+# Description of the project
+description: Federalist is a unified interface for publishing static government websites. It automates common tasks for integrating GitHub, Amazon Web Services and content editing to provide a simple way for developers to launch new websites or more easily manage existing ones.
+
+# Should be 'true' if the project has a continuous build (required)
+# values: true, false
+testable: false
```

Comment:
My reasoning is that there isn't a test suite though there is a continuous build.

@gboone what would you recommend?

> On Sep 4, 2015, at 12:29 PM, Dave Cole <notifications@github.com> wrote:
> 
> In .about.yml:
> 
> > +parent: federalist
> > +
> > +# Maturity stage of the project (required)
> > +# values: discovery, alpha, beta, live
> > +stage: alpha
> > +
> > +# Whether or not the project is actively maintained (required)
> > +# values: active, deprecated
> > +status: active
> > +
> > +# Description of the project
> > +description: Federalist is a unified interface for publishing static government websites. It automates common tasks for integrating GitHub, Amazon Web Services and content editing to provide a simple way for developers to launch new websites or more easily manage existing ones.
> > +
> > +# Should be 'true' if the project has a continuous build (required)
> > +# values: true, false
> > +testable: false
> Should this be true?
> 
> —
> Reply to this email directly or view it on GitHub.
>